lega/utils/storage.py

```diff
@@ -194,7 +194,7 @@ def _fetch(self, start, end, max_attempts=10):
 class S3Storage():
     """S3 object storage and related I/O."""
 
-    def __init__(self, config_section, user):
+    def __init__(self, config_section, bucket):
```

Comment:
This feels strange, can we omit `bucket` from here and place ` bucket = CONF.get_value(config_section, 's3_bucket')` in the section below?

Reply:
Placing it there breaks a bit the usage of the storage function, I did however made it use that configuration value in `verify` and `ingest` so now the archive bucket name is configurable instead of fix value also `FileStorage` archive path is configurable as well.